src/promptflow-azure/promptflow/azure/_utils/_tracing.py

```diff
@@ -67,30 +68,41 @@ def _init_workspace_cosmos_db(init_cosmos_func: typing.Callable) -> None:
 def validate_trace_provider(value: str) -> None:
     """Validate `trace.provider` in pf config.
 
-    1. the value is a valid ARM resource ID for Azure ML workspace
-    2. the workspace exists
-    3. the workspace Cosmos DB is initialized
+    1. the value is a valid ARM resource ID for workspace/project
+    2. the resource exists
+    3. the resource is an Azure ML workspace or AI project
+    4. the workspace Cosmos DB is initialized
     """
-    # valid Azure ML workspace ARM resource ID; otherwise, a ValueError will be raised
+    # valid workspace/project ARM resource ID; otherwise, a ValueError will be raised
     _logger.debug("Validating trace provider value...")
     try:
         workspace_triad = extract_workspace_triad_from_trace_provider(value)
     except ValueError as e:
         raise _create_trace_provider_value_user_error(str(e))
 
-    # the workspace exists
-    _logger.debug("Validating Azure ML workspace...")
+    # the resource exists
+    _logger.debug("Validating resource exists...")
```

Comment:
I feel like the debug info is not really printed when using `--debug` in cli command, could you double confirm that?

Reply:
![image](https://github.com/microsoft/promptflow/assets/38847871/f916a743-11e9-4e6b-ae6f-6d2e3af9d6f3)

i'm not sure, but seems `--verbose` is what you/we need here?

Reply:
both `--verbose` and `--debug` works from my side:

![Uploading image.png…]()